src/test/resources/processor_test_files/2184_CastArithmeticOperand/CastArithmeticOperand.java

```diff
@@ -11,6 +11,7 @@
     long bigNegNum =  Integer.MIN_VALUE-1; // Noncompliant, gives a positive result instead of a negative one.
     int seconds;
     Date myDate = new Date(seconds * 1000); // Noncompliant, won't produce the expected result if seconds > 2_147_483
+    Date myDate2 = new Date(seconds * 1000 * 10); // Noncompliant
```

Comment:
Like we discussed at the meeting, would you mind putting this line in a separate test file, name it something informative, and just put a small comment in the source of that test file to clarify what it's testing?

As the test source files _are_ the test cases, I think we should try to document them as such as well (I've not been great at that myself, but it's time to get better! :)